x/mint/keeper/hooks_test.go

```diff
@@ -39,6 +39,8 @@ func TestEndOfEpochMintedCoinDistribution(t *testing.T) {
 	lastHalvenPeriod := app.MintKeeper.GetLastHalvenEpochNum(ctx)
 	// correct rewards
 	for ; height < lastHalvenPeriod+app.MintKeeper.GetParams(ctx).ReductionPeriodInEpochs; height++ {
+		app.MintKeeper.CreateDeveloperVestingModuleAccount(
+			ctx, sdk.NewCoin("stake", sdk.NewInt(500000)))
```

Comment:
It would be better to only create this vesting account once, with the correct balance.

Then on lines 56/57 we would just test that the balance decreased by the correct amount in that iteration.

Reply:
done